test/utils/throwingInstance.ts

```diff
@@ -0,0 +1,23 @@
+import { instance } from "ts-mockito";
+
+/**
+ * Creates an instance of a mocked object. Calls to methods or properties that have not been stubbed will throw errors
+ */
+export default function throwingInstance<T>(target: T) {
+    const stubbedMethods: Array<string> = Object.keys(
+        (target as any)["tsMockitoInstance"]["mocker"]["methodStubCollections"]
+    );
+
+    const handler = {
+        get: function(target: any, prop: string, receiver: any) {
```

Comment:
The target here can be T, as it turns out you're not using any of the typing in this function. But it's still good practice